pkgs/tools/misc/man-db/default.nix

```diff
@@ -8,24 +9,21 @@ stdenv.mkDerivation rec {
     sha256 = "14p4sr57qc02gfnpybcnv33fb7gr266iqsyq7z4brs6wa6plwrr2";
   };
   
-  buildInputs = [ pkgconfig libpipeline db groff ];
+  buildInputs = [ pkgconfig libpipeline db groff
+                less gzip bzip2 xz ];
   
   configureFlags = [
     "--disable-setuid"
-    "--sysconfdir=/etc"
+    "--sysconfdir=\${out}/etc"
```

Comment:
I dunno ... this seems like a bad idea. First of all, `sysconfdir` most probably defaults to this location anyway, so setting it explicitly is most likely redundant. Secondly, the /nix store is immutable, so there's no way for users to change a configuration file that exists in that place, where the /etc directory is mutable and under the user's control. So expecting the config file in /etc rather than $out/etc actually seems superior.